x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.ts

```diff
@@ -150,7 +192,12 @@ export const getTrustedAppsCreateRouteHandler = (
       const body = getBodyAfterFeatureFlagCheck(req.body, endpointAppContext);
 
       return res.ok({
-        body: await createTrustedApp(exceptionListClientFromContext(context), body),
+        body: await createTrustedApp(
+          exceptionListClientFromContext(context),
+          savedObjectClientFromEndpointContext(endpointAppContext, req),
```

Comment:
yeah, I think that if you just `context.core.savedObjects.client here, you should be good.